docs/advanced_topics/api/v2/usage.md

```diff
@@ -0,0 +1,647 @@
+    -   [Filtering](#filtering)
+    -   [Filtering by tree position (pages only)](#filtering-by-tree-position-pages-only)
+    -   [Search](#search)
+        -   [Search operator](#search-operator)
+    -   [Special filters for internationalised sites](#special-filters-for-internationalized-sites)
+        -   [Filtering pages by locale](#filtering-pages-by-locale)
+        -   [Getting translations of a page](#getting-translations-of-a-page)
+    -   [Fields](#fields)
+        -   [Additional fields](#additional-fields)
+        -   [All fields](#all-fields)
+        -   [Removing fields](#removing-fields)
+        -   [Removing all default fields](#removing-all-default-fields)
+    -   [Detail views](#detail-views)
+    -   [Finding pages by HTML path](#finding-pages-by-html-path)
+-   [Default endpoint fields](#default-endpoint-fields)
+    -   [Common fields](#common-fields)
+    -   [Pages](#pages)
+    -   [Images](#images)
+    -   [Documents](#documents)
+-   [Changes since v1](#changes-since-v1)
+    -   [Breaking changes](#breaking-changes)
+    -   [Major features](#major-features)
+    -   [Minor features](#minor-features)
+
+---
+
+## Fetching content
+
+To fetch content over the API, perform a `GET` request against one of the
+following endpoints:
+
+-   Pages `/api/v2/pages/`
+-   Images `/api/v2/images/`
+-   Documents `/api/v2/documents/`
+
+> The available endpoints and their URLs may vary from site to site, depending
+> on how the API has been configured.
+
+### Example response
+
+Each response contains the list of items (`items`) and the total count
+(`meta.total_count`). The total count is irrespective of pagination.
+
+```
+GET /api/v2/endpoint_name/
+
+HTTP 200 OK
+Content-Type: application/json
+
+{
+    "meta": {
+        "total_count": "total number of results"
+    },
+    "items": [
+        {
+            "id": 1,
+            "meta": {
+                "type": "app_name.ModelName",
+                "detail_url": "http://api.example.com/api/v2/endpoint_name/1/"
+            },
+            "field": "value"
+        },
+        {
+            "id": 2,
+            "meta": {
+                "type": "app_name.ModelName",
+                "detail_url": "http://api.example.com/api/v2/endpoint_name/2/"
+            },
+            "field": "different value"
+        }
+    ]
+}
+```
+
+### Custom page fields in the API
+
+Wagtail sites contain many page types, each with their own set of fields. The
+`pages` endpoint will only expose the common fields by default (such as
+`title` and `slug`).
+
+To access custom page fields with the API, select the page type with the
+`?type` parameter. This will filter the results to only include pages of that
+type but will also make all the exported custom fields for that type available
+in the API.
+
+For example, to access the `published_date`, `body` and `authors` fields
+on the `blog.BlogPage` model in the [:ref:\`configuration docs <apiv2_page_fields_configuration>\`:]()
```

Comment:
This ref looks malformed - please see the documentation in Wagtail's docs for this.

Reply:
I fixed the references based on the [documentation](https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#reference-links)